.travis.yml

```diff
@@ -1,30 +1,30 @@
 sudo: required
-dist: xenial
+dist: bionic
 
   # - git clone -v --progress https://github.com/doronhi/realsense.git  # This is Done automatically by TravisCI
 before_install:
-  - echo 'deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main' || sudo tee /etc/apt/sources.list.d/realsense-public.list
-  # - sudo apt-key adv --keyserver keys.gnupg.net --recv-key C8B3A55A6F3EFCDE || sudo apt-key adv --keyserver hkp://keys.gnupg.net:80 --recv-key C8B3A55A6F3EFCDE
-  - sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main"
+  - sudo apt-key adv --keyserver keys.gnupg.net --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key F6E65AC044F831AC80A06380C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-key C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver keys.gnupg.net --recv-key C8B3A55A6F3EFCDE || \
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C8B3A55A6F3EFCDE
+  - sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo bionic main" -u
   - sudo apt-get update -qq
   - sudo apt-get install librealsense2-dkms --allow-unauthenticated -y
   - sudo apt-get install librealsense2-dev --allow-unauthenticated -y

```

Comment:
Also do we need to add librealsense2 and also perhaps librealsense2-gl ?

Reply:
Basically something similar to https://bitbucket.org/rapyutians/sootballs/src/devel/sensors/app/scripts/install.sh

Reply:
Seems like realsense2_camera does not need it. Maybe they are needed by the utils for actual realsense hardware for display purposes.

Reply:
OK. I see.